core/templates/dev/head/pages/header_js_libs.html

```diff
@@ -12,3 +12,4 @@
 <script src="/third_party/static/jquery-ui-touch-punch-0.3.1/jquery.ui.touch-punch-improved.js"></script>
 <script src="/third_party/static/headroom-js-0.9.4/headroom.min.js"></script>
 <script src="/third_party/static/headroom-js-0.9.4/angular.headroom.min.js"></script>
+<script src="/third_party/static/math-js-4.2.2/math.min.js"></script>
```

Comment:
Hi, is there any specific reason why you include this library in *header_js_libs.html* and not as part of bundle? If not modify the mathJs entry in `manifest.json` and remove this `<script>` tag from here.
```
"mathJs": {
        "version": "4.2.2",
        "downloadFormat": "files",
        "url": "https://cdnjs.cloudflare.com/ajax/libs/mathjs/4.2.2",
        "rootDirPrefix": "math-js-",
        "targetDirPrefix": "math-js-",
        "files": ["math.js", "math.min.js", "math.min.map"],
        "bundle": {
          "js": ["math.min.js"]
        }
      }
```

Reply:
No, there is no specific reason. I'll make these changes.